### DISCUSSION TITLE
simplisafe3 errors, won't arm

### DISCUSSION THREAD

**Discussion by adarodge** [homebridge-simplisafe3/homebridge-simplisafe3#180] [Q&A] [answered]
I've been getting a lot of errors, can you pleas advise me on a resolution so it works again.
Thanks

errors:
Sensor refresh received an error from the SimpliSafe API:
and
Login or refresh failed due to rate limiting or connectivity

**Comment by shamoon** [comment, marked as answer]
Hi @adarodge , as the wording suggests these errors are (mostly) out of our control so there isn’t much we can do about them. Regarding the rate-limiting error, that should usually go away after a few hours or worst case scenario shit down homebridge for a while. Have you set your sensor refresh to something lower than the default 15 seconds? That’s one thing that can get you rate-limited or even blocked by SS (temporarily)
